### PR TITLE
restore framework toggles after a restart

### PR DESCRIPTION
**Pull request by devoncarew**
- after a full restart, restore any framework toggles that the user had set (debug draw, fps overlay, ...). We choose to unset the inspector toggle instead of trying to restore; I think this will mostly match user expectations.

Since we're currently the only client toggling these settings, we don't need events from the framework in order to keep these settings up to date. We should add such long term however.

@jacob314

### CODE REVIEW

**Comment by devoncarew** [comment]
This PR (or a follow-on) may also need to do some work to restore toggles between application runs (or, un-set them, as appropriate for the individual toggle).

**Comment by devoncarew** [comment]
Updated the PR to also handle restoring settings on app launches (if the settings had been toggled on previous runs).

File: src/io/flutter/view/FlutterView.java [hunk 1]
```diff
@@ -336,72 +425,141 @@ public void actionPerformed(AnActionEvent event) {
...
+  }
 }
 
-class RepaintRainbowAction extends AbstractToggleableAction {
+class RepaintRainbowAction extends FlutterViewToggleableAction {
   RepaintRainbowAction(@NotNull FlutterView view) {
     super(view, "Enable Repaint Rainbow");
   }
 
   protected void perform(AnActionEvent event) {
```
Comment (scheglov) [inline src/io/flutter/view/FlutterView.java]: @Nullable ?

File: src/io/flutter/view/FlutterView.java [hunk 2]
```diff
@@ -336,72 +425,141 @@ public void actionPerformed(AnActionEvent event) {
...
 }
 
-class HideSlowBannerAction extends AbstractToggleableAction {
+class HideSlowBannerAction extends FlutterViewToggleableAction {
   HideSlowBannerAction(@NotNull FlutterView view) {
     super(view, "Hide Slow Mode Banner");
   }
 
   @Override
   protected void perform(AnActionEvent event) {
```
Comment (scheglov) [inline src/io/flutter/view/FlutterView.java]: @Nullable  ?

File: src/io/flutter/view/FlutterView.java [hunk 3]
```diff
@@ -336,72 +425,141 @@ public void actionPerformed(AnActionEvent event) {
...
 }
 
-class ShowPaintBaselinesAction extends AbstractToggleableAction {
+class ShowPaintBaselinesAction extends FlutterViewToggleableAction {
   ShowPaintBaselinesAction(@NotNull FlutterView view) {
     super(view, "Show Paint Baselines");
   }
 
   @Override
   protected void perform(AnActionEvent event) {
```
Comment (scheglov) [inline src/io/flutter/view/FlutterView.java]: @Nullable  ?

File: src/io/flutter/view/FlutterView.java (hunk 2 above)
Comment (devoncarew) [reply src/io/flutter/view/FlutterView.java]: done

File: src/io/flutter/view/FlutterView.java (hunk 3 above)
Comment (devoncarew) [reply src/io/flutter/view/FlutterView.java]: done

File: src/io/flutter/view/FlutterView.java (hunk 1 above)
Comment (devoncarew) [reply src/io/flutter/view/FlutterView.java]: done

**Pull request merged**